tests/product/test_connectors.py

```diff
@@ -90,6 +90,22 @@ def test_connector_add_remove_coord_worker_using_dash_h(self):
         self.assert_has_default_connector(self.cluster.master)
         self.assert_has_default_connector(self.cluster.slaves[1])
 
+    def test_connector_add_invalid_host_using_dash_h(self):
```

Comment:
Yes, I have removed these as well. Now, there exists a single unit test that checks for `-H <invalid host name>`